src/gui/symbology/qgsstylemanagerdialog.cpp

```diff
@@ -754,25 +809,18 @@ bool QgsStyleManagerDialog::editColorRamp()
 
 void QgsStyleManagerDialog::removeItem()
```

Comment:
There might be issues when you are trying to remove a mixture of symbols and ramps when you are in the "All" tab.

Reply:
Thanks -- fixed, and code made easier to understand/more robust.